tests/js-modules/modules.py

```diff
@@ -306,10 +288,8 @@ def test_js_execution_time(network, args):
             max_stack_bytes=1024 * 512,
             max_execution_time_ms=1,
         )
-        user_id = "user0"
-        jwt = infra.crypto.create_jwt({"sub": user_id}, jwt_key_priv_pem, jwt_kid)
 
-        r = c.get("/app/jwt", headers={"authorization": "Bearer " + jwt})
+        r = c.post("/app/spin", {"iterations": 123456789})
```

Comment:
```suggestion
        # Do any operation that takes longer than 1ms, confirm it fails
        r = c.post("/app/spin", {"iterations": 123456789})
```

Worth adding a comment here to remind us of the actual goal, since it was a little obscure before.

Reply:
Anyway, let me know if that looks better now